mapper/validator.go

```diff
@@ -0,0 +1,39 @@
+package mapper
+
+import (
```

Comment:
Please organize the imports the following way https://github.com/golang/go/wiki/CodeReviewComments#imports.